app/src/main/java/rectangledbmi/com/pittsburghrealtimetracker/utils/ReactiveHelper.kt

```diff
@@ -0,0 +1,90 @@
+package rectangledbmi.com.pittsburghrealtimetracker.utils
+
+import com.github.pwittchen.reactivenetwork.library.rx2.ReactiveNetwork
+import com.github.pwittchen.reactivenetwork.library.rx2.internet.observing.InternetObservingSettings
+import com.github.pwittchen.reactivenetwork.library.rx2.internet.observing.error.DefaultErrorHandler
+import com.github.pwittchen.reactivenetwork.library.rx2.internet.observing.strategy.WalledGardenInternetObservingStrategy
+import io.reactivex.BackpressureStrategy
+import io.reactivex.Flowable
+import io.reactivex.FlowableTransformer
+import io.reactivex.Observable
+import io.reactivex.functions.Consumer
+import timber.log.Timber
+import java.io.IOException
+import java.net.SocketTimeoutException
+
+/**
+ *
+ * Helper methods for shared reactive methods.
+ *
+ * Created by epicstar on 10/21/16.
+ * @author Jeremy Jao
+ * @since 80
+ */
+
+object ReactiveHelper {
```

Comment:
I wonder if we can turn this into an extension function instead of a static object instead.

Reply:
`isInternetDown` can be an extension function on Throwable, but for `retryIfInternet` I think it's unclear what the receiver should be. I'll just make it a package level function.

Reply:
done.